index.html

```diff
@@ -233,18 +316,31 @@ <h1 class="blog-title">Read Our Latest Posts. Learn Something New Maybe!</h1>
             </div>
             <div class="container">
                 <div class="row">
-                    <div class="btn-toolbar" role="toolbar" aria-label="buttons">
-                        
-                            <button type="button" class="btn btn-default">1</button>
-                            <button type="button" class="btn btn-default">2</button>
-                            <button type="button" class="btn btn-default">3</button>
-                            <button type="button" class="btn btn-default">4</button>
-                      
+                    <div class="btn-toolbar btn-toolbar-gridzilla" role="toolbar" aria-label="buttons">
+                        <button type="button" class="btn btn-default">1</button>
+                        <button type="button" class="btn btn-default">2</button>
+                        <button type="button" class="btn btn-default">3</button>
+                        <button type="button" class="btn btn-default">4</button>
                     </div>
                 </div>
             </div>
         </article>
-
+        <footer>
+            <div class="container">
+                    <div class="row">
+                        <div class="col-sm-3 social-network">
+                            <a href="#" class="t-icon"></a>
+                            <a href="#" class="rss-icon"></a>
+                            <a href="#" class="f-icon"></a>
+                            <a href="#" class="p-icon"></a>
+                        </div>
+                        <div class="col-sm-6 pull-right f-contacts">
+                            <h5>Best PSD Freebies  |  info@bestpsdfreebies.com</h5>
+                            <h6>Copyright 2012 Gridzilla Theme <span>www.bestpsdfreebies.com</span></h6>
```

Comment:
`www.bestpsdfreebies.com` should be a link
`info@bestpsdfreebies.com` - the same